doc/styles/DatePicker.md

```diff
@@ -0,0 +1,55 @@
+# DatePicker Control
+## Styles
+Style Key|IsDefaultStyle*
+-|-
+DatePicker|DatePickerStyle|True
```

Comment:
This renders as
<img width="156" alt="image" src="https://github.com/unoplatform/Uno.Themes/assets/4793020/3118c974-9774-412e-ac38-de97c4c8f594">


As do a few other controls

Style Key should be DatePickerStyle and IsDefaultStyle should just be True, looks like we have "DatePicker" here erroneously, same for ComboBox and a couple others